data_pipeline_api/file_api.py

```diff
@@ -19,15 +19,17 @@ def __init__(
         self._accesses = []
         config_filename = Path(config_filename)
 
-        # Compute a run_id.
-        with open(config_filename, "rb") as file:
-            m = sha1(file.read())
-        m.update(str(self._open_timestamp).encode())
-        self.run_id = m.hexdigest()
-
         try:
             with open(config_filename) as config_file:
                 config = yaml.safe_load(config_file)
+                if "run_id" in config:
+                    self.run_id = config["run_id"]
```

Comment:
Why would we want to set the run_id in the config? Given that you also removed the access log check. Is the idea that we run multiple times under the same id?

Reply:
It may not be necessary, though I have used it in the past so that I can easily locate my output for local runs. I have added a note to the spec about it, which was previously missing.